tests/integration/test_extract.py

```diff
@@ -179,6 +179,97 @@ def test_place_basic(shape, vals):
     assert np.array_equal(arr_np, arr_num)
 
 
+@pytest.mark.xfail(reason="cunumeric raises exception when vals is ndim")
+@pytest.mark.parametrize("vals", VALUES, ids=str)
+@pytest.mark.parametrize("ndim", range(2, DIM), ids=str)
+def test_place_vals_ndim(vals, ndim):
+    shape = (1, 2, 3)
+
+    arr_np = np.random.random(shape)
+    arr_num = num.array(arr_np)
+    mask_np = np.random.random(size=arr_np.size).astype(bool)
+    mask_num = num.array(mask_np)
+    vals_np = np.array(vals, ndmin=ndim).astype(arr_np.dtype)
+    vals_num = num.array(vals_np)
+
+    np.place(arr_np, mask_np, vals_np)
+    num.place(arr_num, mask_num, vals_num)
+    assert np.array_equal(arr_np, arr_num)
```

Comment:
Can you add a comment that elaborates on the differences? c.f. some of the other recent QA/test PRs for good comments in `xfail` cases